sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -54,14 +57,18 @@ def __init__(self, context):
         self.init_ui()
 
     def _detect_hand_id_and_joints(self):
-        self.joint_prefix = None
+        self.joint_prefix = []
         self.hand_joints = None
 
         try:
             joint_states_msg = rospy.wait_for_message("/joint_states", JointState, timeout=1)
-            self.joint_prefix = ([prefix for prefix in joint_states_msg.name if 'h_' in prefix][0]).split("_")[0] + "_"
-            joints = [joint for joint in joint_states_msg.name if self.joint_prefix in joint]
-            self.hand_joints = {self.joint_prefix[:-1]: joints}
+            for prefix in ['rh_', 'lh_']:
```

Comment:
maybe we should have ['rh_', 'lh_'] as a CONST, would make it easier to change if needed.

Reply:
even better a tuple